client/state/sharing/publicize/test/reducer.js

```diff
@@ -25,30 +29,36 @@ import { useSandbox } from 'test/helpers/use-sinon';
 describe( 'reducer', () => {
 	describe( '#fetchingConnections()', () => {
 		it( 'should set fetching to true for fetching action', () => {
-			const state = fetchingConnections( null, {
-				type: PUBLICIZE_CONNECTIONS_REQUEST,
-				siteId: 2916284
-			} );
+			[ PUBLICIZE_CONNECTION_REQUEST, PUBLICIZE_CONNECTIONS_REQUEST ].map( ( type ) => {
```

Comment:
`Array#map` seems unnecessary if you're not using the return value. `Array#forEach` would be more appropriate here (and throughout).